package.json

```diff
@@ -19,10 +19,11 @@
     "@types/node": "^20",
     "@types/react": "^18",
     "@types/react-dom": "^18",
+    "autoprefixer": "^10.4.20",
     "eslint": "^8",
     "eslint-config-next": "14.2.5",
-    "postcss": "^8",
-    "tailwindcss": "^3.4.1",
+    "postcss": "^8.4.41",
```

Comment:
Probably not for this PR, but what does postcss do here? do we need it still?

Reply:
hmm, looks like no, removed it

Reply:
🤩